Source: written by Ilya Barygin <barygin@gmail.com>
Upstream: Unknown, Ubuntu bug #437336
Reason: reason for the patch
--- snack-2.2.10-dfsg1.orig/generic/jkFormatMP3.c
+++ snack-2.2.10-dfsg1/generic/jkFormatMP3.c
@@ -29,6 +29,7 @@
 #include "snack.h"
 #include "jkFormatMP3.h"
 #include <string.h>
+#include <math.h>
 #define FRAS2(is,a) ((is) > 0 ? t_43[(is)]*(a):-t_43[-(is)]*(a))
 #define MAXFRAMESIZE 2106  /* frame size starting at header */
 #define roundf(x) (floor((x)+(float )0.5f))
@@ -1309,8 +1310,6 @@
   return len;
 }
 
-#include <math.h>
-
 #define PI12      0.261799387f
 #define PI36      0.087266462f
 #define COSPI3    0.500000000f
